lib/handleRequest.js

```diff
@@ -17,7 +17,8 @@ function handleRequest (req, res, params, context) {
   var reply = new context.Reply(res, context, request)
 
   if (method === 'GET' || method === 'HEAD') {
-    return handler(reply)
+    handler(reply)
+    return
```

Comment:
Why make this two lines?

Reply:
Because the function does not expect a return value.

Reply:
@jsumners  See https://github.com/fastify/fastify/pull/772#discussion_r167412906